locales/en/translation.json

```diff
@@ -216,32 +216,32 @@
   "hours_timeago": "hours",
   "How do you describe their diabetes?": "",
   "How do you describe your diabetes?": "",
-  "html.basics-no-uploaded-data": "<0>The Basics view shows a summary of your recent device activity, but it looks like you haven't uploaded device data yet.</0><1>To see the Basics, <1>upload</1> some device data.</1><2>If you just uploaded, try <1>refreshing</1>.</2>",
+  "html.basics-no-uploaded-data": "<0>The Basics view shows a summary of your recent device activity, but it looks like your device hasn't sent data yet.</0><1><1>Check</1> your device is activated.</1><2>If you just checked it, try <1>refreshing</1>.</2>",
   "html.browser-warning-download-text": "Or download Chrome <1>here</1>.",
   "html.browser-warning-text": "<0>Copy and paste</0><1></1><2>into Chrome.</2>",
-  "html.donate-form-explainer": " You own your data. Read all the details about Tidepool's Big Data Donation project <1>here</1>.",
+  "html.donate-form-explainer": " You own your data. Read all the details about YourLoops's Big Data project <1>here</1>.",
```

Comment:
A supprimer : nous ne participons pas au Big Data project (que je sache ?)

Reply:
remove ref to Big Data project 731fc2e1